config/mcp/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: flux-operator-mcp
+  labels:
+    app.kubernetes.io/name: flux-operator-mcp
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: flux-operator-mcp
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: flux-operator-mcp
+    spec:
+      serviceAccountName: flux-operator
+      terminationGracePeriodSeconds: 10
+      affinity:
+       nodeAffinity:
+         requiredDuringSchedulingIgnoredDuringExecution:
+           nodeSelectorTerms:
+             - matchExpressions:
+               - key: kubernetes.io/os
+                 operator: In
+                 values:
+                   - linux
+      containers:
+      - name: server
+        image: flux-operator-mcp:latest
+        imagePullPolicy: IfNotPresent
+        args:
+          - serve
+          - --transport=sse
+          - --port=8080
```

Comment:
Maybe we should have `--read-only=true` as the default here and let users write a patch for allowing writes?

Reply:
No, flipping the default from when you run it locally would be super confusing.